tests/lms/views/test_content_item_selection.py

```diff
@@ -1,18 +1,27 @@
+from unittest.mock import Mock
+
 import pytest
 
 from urllib.parse import urlparse
 from lms.exceptions import MissingLTILaunchParamError, MissingLTIContentItemParamError
 from lms.views.content_item_selection import content_item_selection, content_item_form
+from tests.lms.conftest import unwrap
 
+# The `content_item_selection` view function is wrapped in a series of
+# decorators which handle authorization and creating the user/group for the
+# current course.
+#
+# In these tests we only want to test the view function itself, so extract that
+# from the decorated function.
+#
+# nb. This stack of decorators makes it hard to test. It would be much better to
+# use service method calls to handle the work done by decorators.
```

Comment:
Can we remove the last paragraph of this comment? I'm not sure that view body code calling services is actually how we'll refactor this

Reply:
Done